specification/monitor/resource-manager/readme.md

```diff
@@ -55,11 +55,11 @@ input-file:
 - microsoft.insights/preview/2017-11-01-preview/calculateBaseline_API.json
 ```
 
-### Tag: package-2017-08

```

Comment:
You must NOT remove tag, since this may be used by SDK generation and will break it. If you have a strong concern with a tag and you think this should be removed, please start an email thread to the Swagger reviewer alias.